### PR TITLE
Feature/tribes light

### PR DESCRIPTION
**Pull request by androolloyd**
##### Description
Enable Tribes for all organizations

### CODE REVIEW

**Comment by owocki** [comment]
whats the ticket for this?  wuld love to see what tribes lite is
do we plan to port over the existing org tabs to tribes?

**Comment by frankchen07** [comment]
> whats the ticket for this? wuld love to see what tribes lite is
> do we plan to port over the existing org tabs to tribes?

cc @PixelantDesign 

TLDR from a high level:

1. Andrew is working on porting over the "tribe org view" which will deprecate the old organization profile view (so what we can see on gitcoin.co/gitcoinco will be true for all orgs). Unsure if started, I think he was first doing mentors work.

2. In tandem, in discussion with @octavioamu & @PixelantDesign is how we move forward on giving an organization permissions to add users as owners of a gitcoin tribe, so they can post as their organization. This is separate from Github permissions, as detailed by Octavio.

3. Tribes lite: https://docs.google.com/spreadsheets/d/1M3LFchDCofeT-xRdvZDDuTVPvYDpwRZnX19CUVBKul4/edit#gid=0, in short, it's making sure we have the base experiences for sponsors to successfully start and run a hackathon.

**Comment by thelostone-mc** [comment]
Closing this out but reopen if needed
